scheduler/src/cook/config.clj

```diff
@@ -444,12 +444,23 @@
                                   :default-pool "no-pool"}
                                  pool-selection)})))
      :kubernetes (fnk [[:config {kubernetes {}}]]
-                   (merge {:default-workdir "/mnt/sandbox"
-                           :pod-condition-containers-not-initialized-seconds 120
-                           :pod-condition-unschedulable-seconds 60
-                           :reconnect-delay-ms 60000
-                           :set-container-cpu-limit? true}
-                          kubernetes))}))
+                   (let [{:keys [controller-lock-num-shards]
+                          :or {controller-lock-num-shards 32}}
+                         kubernetes
+                         lock-objects
+                         (repeatedly
+                           controller-lock-num-shards
+                           #(Object.))]
+                     (log/info "Doing" controller-lock-num-shards "way lock-sharding in k8s")
+                     (merge {:controller-lock-objects (with-meta
+                                                        lock-objects
+                                                        {:json-value (str lock-objects)})
```

Comment:
What is the metadata for?

Reply:
It's used here:

https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/rest/api.clj#L2678

when producing the JSON output for `/settings`

Reply:
nitpick: change to `(str controller-lock-num-shards " lock-objects")`

Reply:
We're already showing the # of shards in `/settings` with a different key, `controller-lock-num-shards`.